apps/src/lib/config/ethereum.rs

```diff
@@ -8,12 +8,20 @@ pub struct Config {
     /// The Ethereum JSON-RPC endpoint that the Ethereum event oracle will use
     /// to listen for events from the Ethereum bridge smart contracts
     pub oracle_rpc_endpoint: String,
+    /// If this is set to `true`, then instead of the oracle listening for
+    /// events at a Ethereum JSON-RPC endpoint, an endpoint will be exposed by
+    /// the ledger for submission of Borsh-serialized
+    /// [`namada::types::ethereum_events::EthereumEvent`]s
```

Comment:
nit: you can add a `use` statement with the imported path, accompanied by a `#[allow(unused_imports)]` tag. the generated docs reference the full path, with the import this would be shorter.

Reply:
It feels odd importing something just to use it in the docstring, but it makes sense cc https://github.com/rust-lang/rust/issues/79542